manifests/0000_70_dns-operator_03-cluster-operator.yaml

```diff
@@ -1,7 +1,8 @@
+# This resource is not created by CVO, it is only used as a communication
+# mechanism to tell the CVO which ClusterOperator resource to wait for.
+# DNS operator creates and updates this resource.
 apiVersion: config.openshift.io/v1
 kind: ClusterOperator
 metadata:
   name: dns
 spec: {}
```

Comment:
We can delete the `spec` line too.

Reply:
Oh, I see you deleted this earlier and added it back in the latest push.  Do we need it after all? We deleted it in the ingress operator's clusteroperator manifest file.

Reply:
Yes, `spec` is not really needed but this will break hack/release-local.sh
 hack/release-local.sh when it runs `oc apply -f manifests/` will fail with this below error if spec is not present:
```
The ClusterOperator "dns" is invalid: 
...
.spec in body is required
```
We do not really want to create ClusterOpertor resource as part of hack/release-local.sh. I will remove `spec` from this file along with patch to hack/release-local.sh in a followup pr.

Reply:
Thanks for explaining! As far as I can see, the same issue applies to cluster-ingress-operator, so we will want the same change there.

Reply:
Follow up pr: https://github.com/openshift/cluster-dns-operator/pull/105